data/human/human missions.txt

```diff
@@ -5416,3 +5416,346 @@ mission "Deal Change 2"
+				goto fight
+
+			label sidearm
+			`	You try to reach for your pistol, but it's no longer in its holster.`
+
+			label fight
+			`	You stand no chance against the four attackers. They each grab one of your limbs and immobilize you, then the fifth guy points your own gun at you. They must have taken it last night, when you thought they were partying.`
+			`	"Keep it steady, captain," Howard says with glee from your right side. "No funny business. Larry here has an itchy trigger finger." He then lets go of your right leg he has been holding onto, and addresses Larry. "Told you it would work! Next time, I get the gun while you fight in the dirt with the idiot." You notice how none of them seems to be drunk in the slightest anymore.`
+
+			label options
+			`	"You will fly the ship for us. If you don't try anything, we'll let you go afterwards." You don't give much credit to their promise.`
+			choice
+				`	(Pretend to comply with the demands.)`
+					goto comply
+				`	(Shout for your crew.)`
+
+			branch survived
+				random < "flagship crew" - 1
+			`	Larry fires the weapon, and you feel sickening pain in your chest. You fall to the ground, and try to gasp for air, but it feels too hard. You watch as everything goes black and your consciousness slips away.`
+				die
+
+			label survived
+			`	Larry fires the weapon, and you feel sickening pain in your left shoulder. Before Larry could fire again however, a member of your crew pops up behind him and takes him out with a well-aimed shot.`
+			`	With the other four attackers being unarmed, they have no choice but to surrender. As more crew members arrive, they take away the teenagers and stitch up your wound. You still can't quite believe what just happened.`
+			`	You thank your fellow crew members for saving your life, and decide to not let your sidearm out of your sight ever again.`
+				decline
+
+			label comply
+			`	You slowly sit down in the pilot chair, and carefully guide the ship towards the spot Howard indicated on the map. At an opportune moment, you ram the thrusters up to maximum, and turn the ship to the side. Howard and his friends fall over, but you remain secure in your chair.`
+			`	You run over to Larry and make a quick grab for the weapon before he can recover. Now that you have the upper hand, you feel confident to call for your crew.`
+			`	As they take away the five hijackers, the magnitude of the events slowly sinks in. You could easily have died today.`
+			`	You thank your fellow crew members, and decide not to let your sidearm out of your sight ever again.`
+	on complete
+		payment 50000
+		dialog
+			`You notify the spaceport authorities about the situation, and hand over the teenagers to the security detachment they sent out.`
+			`	"That's the 'Howard Gang'! They have been causing trouble on <origin>, then turned tail when they caught wind of the investigation. We are glad you managed to catch them."`
```

Comment:
```suggestion
			`	"That's the 'Howard Gang'! They had been causing trouble on <origin>, then skipped town when they caught wind of the investigation. We are glad you managed to catch them."`
```
Probably a better phrase for the intent here? Tense swap is the other way around this time though